File: app/src/main/java/vergecurrency/vergewallet/view/ui/components/transaction/TransactionItem.java
package vergecurrency.vergewallet.view.ui.components.transaction;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

public interface TransactionItem {
        int getViewType();
        View getView(LayoutInflater inflater, View convertView, ViewGroup parent, int position);
}

File: app/src/main/kotlin/vergecurrency/vergewallet/view/ui/components/transaction/TransactionItem.kt
package vergecurrency.vergewallet.view.ui.components.transaction

import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

interface TransactionItem {
    val viewType: Int
    fun getView(inflater: LayoutInflater, convertView: View, parent: ViewGroup, position: Int): View
}
